setup.cfg

```diff
@@ -74,6 +74,7 @@ install_requires =
     jaraco.windows>=5.7.0; python_version < '3.8' and sys_platform == 'win32'
     scmrepo==0.0.16
     dvc-render==0.0.4
+    dvclive==0.7.2
```

Comment:
Added dvclive as a dependency (https://github.com/iterative/dvc/issues/6527) . 

This is more relevant after these changes to ensure expected behavior

Reply:
Does `dvclive` maintain backward compatibility? I'd prefer it to not be pinned if it is so.

Reply:
> Does dvclive maintain backward compatibility? I'd prefer it to not be pinned if it is so.

At least `>=` is required in order to ensure proper behavior in the future. We might also introduce breaking changes before 1.0`

Reply:
#6527 is unrelated to this tbh. I'd prefer to have it proposed in a separate PR.

Reply:
Also, DVC does not use it, so I think we can just leave it unpinned.

Reply:
Before this change in DVC, "any" DVCLive version could be used alongside DVC. 

However, merging this requires a specific version of DVCLive to work properly. I feel like fixing #6527 is a requirement of this P.R. ?

Reply:
> However, merging this requires a specific version of DVCLive to work properly. I feel like fixing https://github.com/iterative/dvc/issues/6527 is a requirement of this P.R. ?

Sorry, I didn't think of that. Makes sense then.

Reply:
Though I'd still worry about pinning it, as `dvc` users may be using old version of the client, whereas they may want to use recent version of `dvclive`.

Reply:
Included `>=` instead of `==`